modules/collection/programs/fzf.nix

```
@@ -15,15 +20,45 @@ in {
 
     package = mkPackageOption pkgs "fzf" {nullable = true;};
 
+    env = mkOption {
+      type = envVarType;
+      default = {};
+      example = {
+        DEFAULT_COMMAND = "fd --type f";
+        DEFAULT_OPTS = "--layout=reverse --inline-info";
+        CTRL_T_OPTS = ''
+          --walker-skip .git,node_modules,target
+          --preview 'bat -n --color=always {}'
+          --bind 'ctrl-/:change-preview-window(down|hidden|)'
+        '';
+      };
+      description = ''
+        Environment variables passed to fzf with its shell integration.
+
+        Please note that each variable will have `FZF_` prepended to it.
+      '';
+    };
+
     integrations = {
       fish.enable = mkEnableOption "fzf integration with fish";
       zsh.enable = mkEnableOption "fzf integration with zsh";
     };
   };
 
   config = mkIf cfg.enable {
+    assertions = [
+      {
+        assertion = !attrNamesHasPrefix "FZF_" cfg.env;
+        message = ''
+          Each env variable in `rum.programs.fzf.env` is automatically prefixed with `FZF_`. Adding this prefix manually will cause it to be added twice, causing the actual env variable to not apply.
```

```suggestion
          Each env variable in {option}`rum.programs.fzf.env` is automatically prefixed with `FZF_`. Adding this prefix manually will cause it to be added twice, causing the actual env variable to not apply.
```

FWIW doing this will allow newer versions of NDG to automatically link to the relevant option, and format the value (rum.programs.fzf.env) with a special class so it can be themed specifically if desired.

Not a blocker obviously, just wanted to let you know :)